libqtile/window.py

```diff
@@ -990,7 +989,7 @@ def _reconfigure_floating(self, new_float_state=FLOATING):
                 width, height,
                 self.borderwidth,
                 self.bordercolor,
-                above=True,
+                above=above,
```

Comment:
Why adding the `above` argument to the function rather than just switching to False here?

Reply:
As far as I remember, the overall reasoning for adding the parameter at all was that without it moving floating windows around got very expensive (they were layered over and over for every pixel they had been moved).
At that point I didn't dare to mess with this too much, since the effects are a bit unclear. I'll give it a go locally.

Reply:
I am setting this to `False` now. Have to test whether `ScratchPad`s still work properly, though. But they seem to have a few other issues at the moment anyway, so spending some time might be worth it.

Reply:
This seems fine. A pinned window stayed above my `ScratchPad`, in all other cases the `ScratchPad` seemed to win out.